moler/cmd/commandtextualgeneric.py

```diff
@@ -140,6 +141,9 @@ def _build_command_string_escaped(self):
         if self.__command_string is not None:
             if self._max_index_from_beginning != 0 or self._max_index_from_end != 0:
                 sub_command_string = self._build_command_string_slice()
+            elif self._multiline_cmd:
+                cmd_without_new_line = re.sub('\n', '', self.__command_string)
+                sub_command_string = re.escape(cmd_without_new_line)
```

Comment:
What do you think to use slice like single line command? I mean on some terminals the command echo looks really weird.

Reply:
Done